### PR TITLE
print errors in color red

### PR DESCRIPTION
**Pull request by gaochua**
fix for #193

### CODE REVIEW

**Comment by rnehra01** [comment]
I think we can make it a bit more informational/modular by having a common function with different arguments say `INFO`, `ERROR`, `WARNING`.

**Comment by rjt-gupta** [comment]
@rnehra01 I have done some work on this locally. Shall I join?

**Comment by rnehra01** [comment]
Yeah, you can provide suggestions/improvements that can be done.

File: snare/utils/snare_helpers.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 from distutils.version import StrictVersion
 from bs4 import BeautifulSoup
 
+INFO = '\033[97m'  # white
```
Comment (rnehra01) [inline snare/utils/snare_helpers.py]: And you can create a dict to parse the string type.

**Comment by rnehra01** [comment]
@rjt-gupta did you do anything differently?

**Comment by rjt-gupta** [comment]
@rnehra01 I made a colors.py file with info, error, warning as functions. Then used them directly as `colors.error`, `colors.warning`.

**Comment by rnehra01** [comment]
I'm merging this PR. @afeena Do you think of a better name for the method `print_color`?

File: snare/utils/snare_helpers.py [hunk 2]
```diff
@@ -103,3 +103,16 @@ def str_to_bool(v):
         return False
     else:
         raise argparse.ArgumentTypeError('Boolean value expected')
+
+
+def print_color(msg, mode='INFO'):
+    colors = {
+        'INFO': '\033[97m',  # white
+        'ERROR': '\033[31m',  # red,
```
Comment (rnehra01) [inline snare/utils/snare_helpers.py]: Remove the comma in the end.

**Pull request merged**